engine/battle_anims/anim_commands.asm

```diff
@@ -162,7 +162,7 @@ BattleAnimRestoreHuds:
 	pop af
 	ldh [rSVBK], a
 
-	ld a, $1
+	ld a, 1
```

Comment:
The various loads/reads of `hBGMapMode` should use constants eventually, so might as well leave them as hex until then.